packages/untp-test-suite-mocha/src/inferences/10-infer-product-claim-criteria-verified.n3

```diff
@@ -0,0 +1,57 @@
+@prefix dcc: <https://test.uncefact.org/vocabulary/untp/dcc/0/> .
+@prefix dpp: <https://test.uncefact.org/vocabulary/untp/dpp/0/> .
+@prefix log: <http://www.w3.org/2000/10/swap/log#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
```

Comment:
I'm assuming these inference files are just the same those in the other repo (with any small changes due to changes in the DPP etc. format?).

Reply:
Yes, the only difference is that in https://github.com/uncefact/tests-untp/commit/b0d381ac775c153911ec89e66cd7abf079582ffb i have changed `untp:Criterion` to `untp:assessmentCriteria` according to the latest json-ld context.